Tests.Integration.Presentation.Web/UI_Configuration/UIConfigurationTests.cs

```diff
@@ -51,11 +47,103 @@ public async Task Can_Get()
             using var response = await UIConfigurationHelper.SendGetRequestAsync(TestEnvironment.DefaultOrganizationId, module);
 
             //Assert
-            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            Assert.Equal(HttpStatusCode.NotFound ,response.StatusCode);
+        }
+
+        [Fact]
+        public async Task Get_Returns_Forbidden_If_Not_Allowed_To_Read()
+        {
+            //Arrange
+            var module = A<string>();
+            var (cookie, organization) = await CreatePrerequisitesAsync();
+            var (cookieUser2, _) = await CreatePrerequisitesAsync();
+
+            await UIConfigurationHelper.CreateUIModuleAndSaveAsync(organization.Id, module, cookie);
 
-            var data = await response.ReadResponseBodyAsKitosApiResponseAsync<UIModuleCustomizationDTO>();
-            Assert.Empty(data.Nodes);
-            Assert.NotNull(data.Nodes);
+            //Act
+            using var response = await UIConfigurationHelper.SendGetRequestAsync(organization.Id, module, cookieUser2);
+
+            //Assert
+            Assert.Equal(HttpStatusCode.Forbidden ,response.StatusCode);
+        }
+
+        [Fact]
+        public async Task Put_Deletes_Nodes()
```

Comment:
make sure to test the case where you also do changes to existing nodes perhaps just take the existing nodes and flip all of the "enabled" states.